src/Services/Podcasts/Podcast.Infrastructure/Data/Models/UserSubmittedFeed.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Podcast.Infrastructure.Data.Models;
+
+public class UserSubmittedFeed
```

Comment:
Could this be a `record`, does EF support that?

Reply:
i think not recommended at this time from my understanding.